go-update/src/test/java/org/reactome/release/goupdate/JavaVsPerlComparison.java

```diff
@@ -0,0 +1,188 @@
+package org.reactome.release.goupdate;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import org.gk.model.GKInstance;
+import org.gk.model.InstanceUtilities;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.SchemaAttribute;
+import org.reactome.util.compare.DBObjectComparer;
+
+public class JavaVsPerlComparison {
+
```

Comment:
Will this test be needed moving forward? I assume after the transitionary release where we first go from Perl to Java that this test will be 'pending obsoletion'

Reply:
Yeah, after a few releases that use this code, after it's trusted, then this test code won't really be needed anymore.